### DISCUSSION TITLE
"PROBING_HEATERS_OFF"  not working while UBL Probing from menu

### DISCUSSION THREAD

**Discussion by xantrk** [Jyers/Marlin#322] [Ideas]
Hello Jyers, 
Thank you once again for the firmware. 
As my bltouch clone arrived and tries to crash the bed a couple times already, I have seen some flags in Marlin config such as:

PROBING_HEATERS_OFF       // Turn heaters off when probing
 PROBING_FANS_OFF          // Turn fans off when probing

with a note: "* Heaters and/or fans can be disabled during probing to minimize electrical
 * noise. A delay can also be added to allow noise and vibration to settle.
 * These options are most useful for the BLTouch probe, but may also improve
 * readings with inductive probes and piezo sensors."

This really helps while using G29 command and tilting or other ABL methods that is called form GCODE I tried, no crashes to bed. Can you possibly implement this behaviour to leveling menu?

It turns the heater and fans of as probe approaches, and re-actives while going up until  next descent.

**Comment by Jyers** [comment]
Unfortunately if this is not implemented into the UBL gcode there is nothing I can do to implement it from the menu. If you believe this is an issue with UBL you should post an issue report on the upstream branch to get it sorted out. Sorry I can't be of more help.
